clients/flash/air-client/src/org/bigbluebutton/air/common/views/skins/MobileToggleSwitchSkin.as

```diff
@@ -0,0 +1,382 @@
+package org.bigbluebutton.air.common.views.skins {

```

Comment:
If you take their code you can't just remove the license from it and slap ours on. I believe you can say the original code falls under ASF and modifications fall under LGPL, but I'm not 100% sure. The best option is to just not copy their code directly though.